frontend/src/app/libs/container/components/localization/localization.component.html

```diff
@@ -4,9 +4,9 @@
 	</app-button>
 	<mat-menu class="localization__menu" #menu="matMenu">
 		<button mat-menu-item class="localization__menu-item"
-				*ngFor="let locale of translateService.getLangs()"
-				(click)="translateService.use(locale)">
-			<span>{{locale | uppercase}}</span>
+			*ngFor="let locale of translateService.getLangs()" (click)="translateService.use(locale)">
+		<img [src]=" locale == 'en' ? 'assets/images/localization/icon-uk-flag.svg' : 'assets/images/localization/icon-russian-flag.svg'" class="localization__flag">
+			<span class="localization__language">{{(locale == 'en' ? 'BUTTON.LANGUAGE-EN' : 'BUTTON.LANGUAGE-RU') | translate}}</span>
```

Comment:
move `locale == 'en' ? 'BUTTON.LANGUAGE-EN' : 'BUTTON.LANGUAGE-RU') ` to getter

Reply:
use ===

Reply:
Agree with you, but also can suggest to use map pattern to get locale metadata.
In result just need to use smth like getLocaleMetadata(locale).buttonLanguage or getLocaleMetadata(locale).iconUrl.
And map can be organized in the following way: locale as key, metadata object as value.
What do you think about that?